docs/book/cookbook/aot-guide.md

```diff
@@ -54,7 +50,10 @@ $ composer require --dev zendframework/zend-code
 > $ composer require zendframework/zend-di:^3.0
 > ```
 >
-> This approach will also notify you if there are conflicts with installing v3.
+> This approach will also notify you if there are conflicts with installing v3. 
+
+> __Note:__ Before version 3.1 of `zend-di`, `zendframework/zend-code` was required 
```

Comment:
For a notice we use a headline with the order of 3 and meaningful wording. Like:

https://github.com/zendframework/zend-di/blob/fb2468c5981109aaed2a3d7367211f02f2449702/docs/book/cookbook/aot-guide.md#manually-creating-a-generator-instance